cmake/toolchains/ios.toolchain.cmake

```diff
@@ -0,0 +1,632 @@
+#
```

Comment:
@ Thor Is this the feature-reduced toolchain that's only got the targets we care about in it?

Reply:
Yes correct! I removed all simulator, watchos, tvOS and some features we were not using - it's quite simplified, could probably in theory be even smaller but was at the same time somewhat fond of how "tested" it is and seems to care about some common bugs faced